NEWS

```diff
@@ -1,4 +1,8 @@
 DFHack Future
+    New Scripts
+		hotkey-notes:  print key, name, and jump position of hotkeys
+    Removed
+		needs_porting/*
 
```

Comment:
Could you use spaces here? Tabs and spaces generally don't mix very well, as evidenced here, and the rest of this file uses spaces.